QUEST.tsv

```diff
@@ -64,53 +64,53 @@ xQUEST_20150317_000063	What on earth do you call this?
 xQUEST_20150317_000064	I think the Goddess of Earth - Zemyna would mourn more than anyone else.
 xQUEST_20150317_000065	The Pokubu have always been here, but aside from them, there's been the occasional monster.
 xQUEST_20150317_000066	I've thought about it. It's no mistake that the goddesses have left us all. {nl}If that weren't so, then there wouldn't be so many monsters.
-xQUEST_20150317_000067	They say there's a place in the forest where someone earned a favor from the goddesses…
+xQUEST_20150317_000067	They say there's this place in the forest where someone bought a favor from the goddesses…
```

Comment:
The Korean says "They say there's a person who lives in this forest who received strength from the goddesses.."